### DISCUSSION TITLE
Help needed with games

### DISCUSSION THREAD

**Discussion by sudousercom** [eternnoir/pyTelegramBotAPI#1746] [Q&A]
Hi! Any searches with no luck, maybe someone can explain, how can i start game from an url, not embed in python,  i have http://site.com/game, game added to bot father, i have some command to start it, and inline query for game, but i dont understand, how can i run it in telegram clients. Thanks!

**Comment by coder2020official** [comment]
Check this:
https://core.telegram.org/bots/games

**Reply by sudousercom** [reply]
I already read this link, one of the first, but I still don’t understand when and how I need to send the user a link to the game so that it starts in the built-in telegram browser of the client, by default the game is sent with a button that does nothing , and it says that I can use it, or make my own, but which one, and how to pass the parameters, I don’t understand

**Reply by coder2020official** [reply]
https://pytba.readthedocs.io/en/latest/sync_version/index.html#telebot.TeleBot.send_game

**Reply by sudousercom** [reply]
so, i need to use this method, but it gives me an error about inline keyboard, it must be an json encoded, and i cant find any examples, and what i need add in this button?
